rules-reviewed/eap7/eap71/hibernate51-53.windup.xml

```diff
@@ -224,17 +224,24 @@
                 </hint>
             </perform>
         </rule>
-        <!--https://issues.jboss.org/browse/WINDUPRULE-381
+        <!--https://issues.jboss.org/browse/WINDUPRULE-381 -->
         <rule id="hibernate51-53-00500">
             <when>
+                <javaclass references="org.hibernate.{className}" >
+                </javaclass>
             </when>
             <perform>
+                <hint title="Hibernate 5.3 - RegionFactory usage" effort="2" category-id="mandatory">
```

Comment:
It seems to me `3` is better in this case (refer to [A.1.2. How Story Points are Estimated in Rules](https://access.redhat.com/documentation/en-us/red_hat_application_migration_toolkit/4.2/html-single/rules_development_guide/#how_story_points_are_estimated_in_rules))
```suggestion
                <hint title="Hibernate 5.3 - RegionFactory usage" effort="3" category-id="mandatory">
```